File: frontend/src/shared/enums/backend/flow-search-area-type.enum.ts
export const FlowSearchAreaTypeEnum = {
  CUSTOM: "CUSTOM",
  APPLICATION: "APPLICATION",
  BROWSER_TAB: "BROWSER_TAB",
  MONITOR: "MONITOR",
} as const;

export type FlowSearchAreaTypeEnum =
  (typeof FlowSearchAreaTypeEnum)[keyof typeof FlowSearchAreaTypeEnum];

File: backend/Core/Enums/FlowSearchAreaTypeEnum.cs
namespace Core.Enums
{
    public enum FlowSearchAreaTypeEnum
    {
        CUSTOM,       // A rectangle, optionally inside another area
        APPLICATION,  // A window, found at runtime
        BROWSER_TAB,  // A browser tab's document area, found through UI Automation
        MONITOR,
    }
}
